spec/models/course/condition/assessment_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe Course::Condition::Assessment, type: :model do
+  it { is_expected.to act_as(:condition).class_name(Course::Condition.name) }
+
+  let(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+
+    describe '#title' do
+      context 'when there is no minimum grade percentage' do
+        it 'returns the correct title' do
+          subject.assessment = create(:course_assessment_assessment, title: 'Dummy', course: course)
+          expect(subject.title).to eq('Dummy')
+        end
+      end
+
+      context 'when there is minimum grade percentage' do
+        it 'returns the correct title' do
+          pending('Assessment#title uses number_to_percentage which uses I18 but we change I18
```

Comment:
We changed the I18n in test environment to return the key instead of the value. I use `number_to_percentage` method which uses I18n and thus it doesn't work in test environment.